src/views/Signup/index.vue

```diff
@@ -191,7 +191,8 @@ export default {
 
       const payload = {
         email: this.email,
-        password: sha256(this.initialPassword)
+        password: sha256(this.initialPassword),
+        redirect: 'https://app.storj.io/'
```

Comment:
This may not work in non production environments. We should pull the url and proto and remove everything after to get this.

Reply:
This has little relevance outside of production because it only shows up in the confirmation email.

Reply:
Good point, agreed. Merging.